### DISCUSSION TITLE
How to set verbosity in configuration file?

### DISCUSSION THREAD

**Discussion by elchenberg** [PHPCSStandards/PHP_CodeSniffer#617] [Q&A] [answered]
Is there a way to set the verbosity in a configuration file?

I tried `<arg value="v"/>` (and `vv` and `vvv`) but it is ignored.

I tried `<arg value="-v"/>` (and `-vv` and `-vvv`) but I get the error `ERROR: option "--v" not known`.

I tried `<arg name="v"/>` (and `vv` and `vvv`) but I get the error `ERROR: option "--v" not known`.

I tried `<arg name="verbose"/>` knowing that this option does not exist and I get the error `ERROR: option "--verbose" not known`.

**Comment by jrfnl** [comment, marked as answer]
Not as far as I know. This is documented as such as well:
```xml
 <!--
    You can hard-code command line values into your custom standard.
    Note that this does not work for the command line values:
    -v[v][v], -l, -d, --sniffs and --standard
    
    The following tags are equivalent to the command line arguments:
    --extensions=php,inc --report=summary --colors -sp
 -->
 <arg name="extensions" value="php,inc" />
 <arg name="report" value="summary"/>
 <arg name="colors"/>
 <arg value="sp"/>
```

Source: https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Annotated-Ruleset

Out of curiousity: why would you want to always run in `verbose` mode ? It's intended as a debugging feature, not for standard use.

**Reply by elchenberg** [reply]
@jrfnl I do not know. :sweat_smile:
